DH_Interface/Classes/DHContextMenu_SquadMembers.uc

```diff
@@ -173,6 +188,11 @@ protected function ProcessEntry(int EntryIndex, GUIComponent Component)
             PC.ServerSquadInvite(SelectedPRI);
             return;
 
+        case 15:
```

Comment:
The way this work is confusing, but I'll allow it since this code will get refactored in the future.

Reply:
Fixed in 457bbad6d4ebfdeb73f232baa6b21c055c46fc1a.